src/Services/Gateways/Pix.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Ebanx\Benjamin\Services\Gateways;
+
+use Ebanx\Benjamin\Models\Country;
+use Ebanx\Benjamin\Models\Currency;
+use Ebanx\Benjamin\Models\Payment;
+use Ebanx\Benjamin\Services\Adapters\PixPaymentAdapter;
+
+class Pix extends DirectGateway
+{
+    const API_TYPE = 'pix';
+
+    protected static function getEnabledCountries()
+    {
+        return [Country::BRAZIL];
+    }
+
+    protected static function getEnabledCurrencies()
+    {
+        return [
+            Currency::BRL,
+            Currency::USD,
+            Currency::EUR,
```

Comment:
Hi @netojoaobatista, I suggested you remove these currencies for a while because I guess we can't handle pix in EUR and USD. Just avoid errors in our monitor system if someone tries to use these currencies.